api/endpoint-tests/endpoint-coverage.js

```diff
@@ -9,6 +9,17 @@ const readline = require('readline');
 // strip out the undefineds, since those don't survive stringing
 const openAPI = JSON.parse(JSON.stringify(require('../routes/openAPI')));
 
+const getOpenApiUrl = url => {
+  let last = url.replace(/\/:(.+?)(\/|$)/g, '/{$1}$2');
+  let out = last.replace(/\/:(.+?)(\/|$)/g, '/{$1}$2');
+
+  while (out !== last) {
+    last = out;
+    out = last.replace(/\/:(.+?)(\/|$)/g, '/{$1}$2');
+  }
+  return out;
+};
+
```

Comment:
The previous regex was only catching the first `/:variable:/` in a URL, so something like `/:var1:/:var2:` wasn't getting caught.  The endpoint "coverage" report was messed up, as a result.